bundles/framework/featuredata/plugin/FeatureDataPluginHandler.js

```diff
@@ -170,18 +193,148 @@ class FeatureDataPluginUIHandler extends StateHandler {
         };
     }
 
+    createSelectByPropertiesSettings (features) {
+        const settings = this.createVisibleColumnsSettings(features);
+        let filters;
+        if (settings?.allColumns && settings?.allColumns.length) {
+            filters = [this.initEmptyFilter(settings.allColumns[0])];
+        }
+        return settings?.allColumns ? { allColumns: settings.allColumns, filters, errors: null } : null;
+    }
+
     closeFlyout (resetLayers) {
         if (this.flyoutController) {
             this.flyoutController.close();
             this.flyoutController = null;
         }
-        this.updateState({ flyoutOpen: false, layers: resetLayers ? null : this.getState().layers });
+
+        if (this.popupController) {
+            this.popupController.close();
+            this.popupController = null;
+        }
```

Comment:
```suggestion
       this.closePopup();
```
Or the renamed version.